src/kedro_templar/plugin.py

```diff
@@ -0,0 +1,23 @@
+import click
+from . import process_templates as pt
+import logging
+
+log = logging.getLogger(__name__)
+
+
+@click.group(name="JSON")
```

Comment:
can we change this name to something else than `JSON`?

Reply:
sure:) yml is  ok?